exciting-kfs/kfs
# feat: reading partition tables

- test: write DMA
- feat: read partition
  - make and equip a HDD divided into two partitions with the ext2 filesystem installed.
  - read the MBR in each device and parse partition tables.
- fix: read sectors(pio)
  - check status register before reading data port.
